mail_tracking/__openerp__.py

```diff
@@ -5,7 +5,7 @@
 {
     "name": "Email tracking",
     "summary": "Email tracking system for all mails sent",
-    "version": "8.0.3.0.0",
+    "version": "8.0.3.0.1",
```

Comment:
Thanks for the pointer, although again it says that backwards-compatible new features increment `y`. This is the case.

However I'm not going to insist any more on this, it's not worth the effort, just do as you please :wink: